qa/c8-orchestration-cluster-e2e-test-suite/pages/OperateProcessesPage.ts

```diff
@@ -105,6 +123,22 @@ class OperateProcessesPage {
       }),
     ).toBeVisible();
   }
+
+  async clickStartDateSortButton(): Promise<void> {
+    await this.startDateSortButton.click();
+  }
+
+  async clickProcessInstanceKeySortButton(): Promise<void> {
+    await this.processInstanceKeySortButton.click();
+  }
+
+  async clickVersionSortButton(): Promise<void> {
+    await this.versionSortButton.click();
+  }
+
+  async clickProcessNameSortButton(): Promise<void> {
+    await this.processNameSortButton.click();
+  }
```

Comment:
We don't need to create a method for each action, just using `this.processNameSortButton.click()` in the test is enough